typing/ctype.ml

```diff
@@ -814,27 +814,24 @@ let rec normalize_package_path env p =
 
 let rec check_scope_escape env level ty =
   let ty = repr ty in
-  if not_marked_node ty then begin
-    if level < ty.scope then
+  let orig_level = ty.level in
+  if try_logged_mark_node ty then begin
+    if orig_level < ty.scope then
```

Comment:
I think this should be `level < ty.scope`, not `orig_level`, right?

Reply:
It should indeed... Thanks, fixed.